packages/synapse-react-client/src/lib/containers/auth/TwoFactorBackupCodes.tsx

```diff
@@ -81,96 +81,95 @@ export default function TwoFactorBackupCodes(props: TwoFactorBackupCodesProps) {
-                icon={false}
-              >
-                <strong>These new codes have replaced your old codes.</strong>{' '}
-                Please save these codes and use them from now on.
-              </Alert>
-            )}
-            <RecoveryCodeGrid recoveryCodes={recoveryCodes} />
-            {error && (
-              <Alert severity={'error'} sx={{ my: 2 }}>
-                {error.message}
-              </Alert>
-            )}
-            <Box
-              sx={{
-                display: 'grid',
-                gridTemplateColumns: '1fr 1fr',
-                gap: '10px',
-                mb: '10px',
-                ...hideOnPrintStyle,
-              }}
+              <strong>These new codes have replaced your old codes.</strong>{' '}
+              Please save these codes and use them from now on.
+            </Alert>
+          )}
+          <RecoveryCodeGrid recoveryCodes={recoveryCodes} />
+          {error && (
+            <Alert severity={'error'} sx={{ my: 2 }} icon={false}>
+              {error.message}
+            </Alert>
+          )}
+          <Box
+            className={hiddenOnPrintClassName}
+            sx={{
+              display: 'grid',
+              gridTemplateColumns: '1fr 1fr',
+              gap: '10px',
+              mb: '10px',
+            }}
+          >
+            <Button
+              variant={'outlined'}
+              onClick={copyCodesToClipboard}
+              disabled={!recoveryCodes}
             >
-              <Button
-                variant={'outlined'}
-                onClick={copyCodesToClipboard}
-                disabled={!recoveryCodes}
-              >
-                Copy to Clipboard
-              </Button>
-              <Button
-                variant={'outlined'}
-                disabled={!recoveryCodes}
-                onClick={window.print}
-              >
-                Print Codes
-              </Button>
-            </Box>
+              Copy to Clipboard
+            </Button>
             <Button
-              variant={'contained'}
-              fullWidth
+              variant={'outlined'}
               disabled={!recoveryCodes}
-              sx={hideOnPrintStyle}
-              onClick={onClose}
+              onClick={window.print}
             >
-              Done
+              Print Codes
             </Button>
           </Box>
-        </>
+          <Button
+            variant={'contained'}
+            className={hiddenOnPrintClassName}
+            fullWidth
+            disabled={!recoveryCodes}
+            onClick={onClose}
+          >
+            Done
+          </Button>
+        </Box>
       }
       rightContent={
         <Box
           sx={{
             py: 10,
             height: '100%',
-            background: `url(${TwoFactorGraphic}) no-repeat right bottom`,
+            background: `url(https://s3.amazonaws.com/static.synapse.org/images/twofactor-graphic.svg) no-repeat right bottom`,
```

Comment:
SVGR is configured to let us load SVGs as React components. At this time, our esbuild config does not let us load them as dataurls, so the previous code didn't work.